include/cuco/static_map.cuh

```diff
@@ -1350,16 +1359,21 @@ class static_map {
   /**
    * @brief Gets the number of elements in the hash map.
    *
+   * @param stream Stream used for size computation
+   *
    * @return The number of elements in the map
    */
-  std::size_t get_size() const noexcept { return size_; }
+  std::size_t get_size(cudaStream_t stream = 0) const noexcept;
```

Comment:
```suggestion
  [[nodiscard]] std::size_t get_size(cudaStream_t stream = 0) const noexcept;
```

Reply:
Resolved, since the missing [[nodiscard]] attribute is on-purpose to not break with existing downstream projects.